### ISSUE TITLE
Advertisment blocking players tetris view 

### ISSUE THREAD

**Issue by EloiseYan** [o5k/tetrio-issues#1315]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this issue is not already reported.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am using a **supported browser** or the Desktop Client.
- [X] I am not using anything that **modifies game code** (e.g. TETR.IO PLUS).
- [X] I am certain this is a **bug**, not a feature request.

### OS and Version

Windows 11

### Browser and version

version23H2

### Describe the bug/issue

The advertisment block half of the screen and i cant play the game nicely as my view has been blocked by the advertisment in tetrio

### Reproducing

Go to tetr.io
play multiplayer
see error

### Expected behavior

You can put any advertisment anyplace you like as long it dont block the view of the player in game

### Screenshots

.

### Console logs

.

### Additional context

.

**Comment by ZaptorZap** [comment]
Duplicate of https://github.com/tetrio/issues/issues/1122
